tours/tcga/data.R

```diff
@@ -11,20 +11,20 @@ eh1 <- ehub[["EH1"]] # an ExpressionSet
 se1 <- as(eh1, "SummarizedExperiment")
 sce <- as(se1, "SingleCellExperiment")
```

Comment:
Suggest renaming `exprs` to `counts` in the assays, if that's what they really are.